raspimouse/src/raspimouse_component.cpp

```diff
@@ -123,8 +125,10 @@ CallbackReturn Raspimouse::on_configure(const rclcpp_lifecycle::State &)
       &Raspimouse::publish_switches, this));
   switches_timer_->cancel();
   // Timer for publishing light sensor information
+  declare_parameter(LIGHT_SENSORS_HZ_PARAM, 10.0);
+  std::chrono::milliseconds ls_duration{static_cast<long int>(1000.0/get_parameter(LIGHT_SENSORS_HZ_PARAM).get_value<double>())};
   light_sensors_timer_ = create_wall_timer(
-    100ms, std::bind(
+    ls_duration, std::bind(
```

Comment:
変数名を略さないほうが、読みやすいコードになります。
```suggestion
    light_sensors_duration, std::bind(
```